src/main/untangled/ui/elements.cljc

```diff
@@ -126,6 +127,91 @@
                        (dissoc :color))]
     (apply dom/div (clj->js props) children)))
 
+(defn ui-avatar
+  "Render an icon or a short string within an avatar. Normal HTML/React attributes can be included, and should be a cljs map (not a js object).
+
+  color (optional): :primary, :accent
+  size (optional): huge"
+  [{:keys [className color size] :as props :or {className ""}} child]
+  (let [legal-colors #{:primary :accent}
+        legal-sizes #{:huge}
+        user-classes (get props :className "")
+        classes (cond-> user-classes
+                        :always (str " c-avatar")
+                        (contains? legal-colors color) (str " c-avatar--" (name color))
+                        (contains? legal-sizes size) (str " c-avatar--" (name size)))
+        props (cond-> props
+                      :always (assoc :className classes)
+                      :always (dissoc :color)
+                      :always (dissoc :size))]
+    (dom/span (clj->js props) child)))
+
+(defn ui-loader
+  "Render an icon or a short string within an avatar. Normal HTML/React attributes can be included, and should be a cljs map (not a js object).
+
+  color (optional): :neutral"
+  [{:keys [className color] :as props :or {className ""}}]
+  (let [legal-colors #{:neutral}
+        user-classes (get props :className "")
+        classes (cond-> user-classes
+                        :always (str " c-loader")
+                        (contains? legal-colors color) (str " c-loader--" (name color)))
+        props (cond-> props
+                      :always (assoc :className classes)
+                      :always (dissoc :color))]
+    (dom/div (clj->js props))))
+
+(defn ui-icon
+  "Render an icon. Normal HTML/React attributes can be included, and should be a cljs map (not a js object).
+
+  color (optional): :positive, :informative, :neutral, :live, :alterable, :negative
+  size (optional): :small, :medium, :large, :xlarge, :huge"
+  [{:keys [className color size] :as props :or {className ""}} child] ;TODO: SHould we allow children?
```

Comment:
Currently it only supports 1, but the plan is to let it handle up to 4 images that will layout out in a miniature inline grid for things like group chat.